shell/device_sdk_c_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex -o pipefail
+
+echo "--> device_sdk_c_build.sh"
+
+UID=`id -u`
+GID=`id -g`
+
+mkdir $WORKSPACE/release
+
+DOCKER_BUILD_CONTAINER=${DOCKER_NAME}-builder-${BUILD_ID}
+
+echo "Starting build container and generating tar.gz"
+# --privileged is required because for some reason when docker bind mounts the
+# on the workspace on the Jenkins build server, it is Read-Only
+docker run --rm -e UID -e GID --privileged -v $WORKSPACE/release:/edgex-c-sdk/build/release ${DOCKER_IMAGE}
```

Comment:
Should `UID` and `GID` be `$UID` and `$GID`?

Reply:
No, you dont have to do that with docker. Docker will automatically infer environment variables for you so you dont have to do things like: -e UID=$UID. This has been tested on the sandbox. https://jenkins.edgexfoundry.org/sandbox/job/device-sdk-c-v2-master-verify-docker/